### ISSUE TITLE
D2LTESTS004 failure on an abstract test fixture base

### ISSUE THREAD

**Issue by mthjones** [Brightspace/D2L.CodeStyle#879]
When adding an abstract test fixture base class that contains tests to a test framework project, D2LTESTS004 analyzer errors are reported on the test, despite the class itself not being marked with `[TestFixture]`.

e.g. Putting this in a project marked as a test framework will fail the analyzer.

```cs
public abstract class FooBase {
    [Test]
	public void FooTest() {}
}
```

r? @omsmith
